projects/plugins/crm/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.3.1] - 2023-12-14
+### Security
```

Comment:
Is this something we've historically done in CRM, having a Security heading? Normally it's just been Improved, Added or Fixed.

Reply:
I noticed it, but left it there because in my opinion it looked nice. I will post this in the thread to see what others think. 

Thanks for pointing it out.